mkosi/__init__.py

```diff
@@ -6052,6 +6052,50 @@ def virt_name(args: CommandLineArguments) -> str:
     return cast(str, name[:13])
 
 
+def has_networkd_vm_vt() -> bool:
+    networkd_search_paths = run(["systemd-path", "systemd-search-network"], stdout=PIPE, text=True).stdout
+    for path in networkd_search_paths.split(":"):
```

Comment:
This may be a stupid nit, but
```suggestion
    networkd_search_paths = run(["systemd-path", "systemd-search-network"], stdout=PIPE, text=True).stdout.split(":")
    for path in networkd_search_paths:
```